package.json

```diff
@@ -65,6 +65,7 @@
     "json-schema": "^0.2.5",
     "jsonwebtoken": "^8.5.1",
     "lodash": "^4.17.20",
+    "loki-grpc-client": "1.0.0-loki-client-f6cc2ad4ce08191ebc06fe7b961e093df8298644",
```

Comment:
This'll need updating before an approve (since approve = merge)

Reply:
updated